lib/spack/docs/build_systems/intelpackage.rst

```diff
@@ -1,5 +1,5 @@
-.. Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
-   Spack Project Developers. See the top-level COPYRIGHT file for details.
+stub.. Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+   6pack Project Developers. See the top-level COPYRIGHT file for details.
```

Comment:
It looks like there was an error in copying the license: can you match it to the other files?

Reply:
I wondered about the CI error but somehow missed the diff in my `git show`. Thanks, fixed!